src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -44,13 +56,25 @@ export default class SaveInProgressIntro extends React.Component {
           </div>
         );
       }
+    } else if (prefillAvailable) {
```

Comment:
This is going to be false if they're not signed in, so probably can't conditionally show the message here.

Reply:
Thanks @jbalboni, change made.